lms/templates/courseware/progress.html

```diff
@@ -81,7 +84,7 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
                         <p class="copy">${_("You can keep working for a higher grade, or request your certificate now.")}</p>
                     </div>
                     <div class="msg-actions">
-                        <button class="btn" data-endpoint="${reverse('generate_user_cert', args=[unicode(course.id)])}" id="btn_generate_cert">${_('Request Certificate')}</button>
+                        <button class="btn generate_certs" data-endpoint="${reverse('generate_user_cert', args=[unicode(course.id)])}" id="btn_generate_cert">${_('Update Certificate')}</button>
```

Comment:
This looks like the "Request Certificate" button (the one the user clicks if he hasn't already requested a certificate), so it should stay the same - ? The one that needs to change to "Update Certificate" is the "Regenerate Certificate" button (line 73), correct?